include/GafferSceneUI/StandardLightVisualiser.h

```diff
@@ -71,7 +71,7 @@ class GAFFERSCENEUI_API StandardLightVisualiser : public LightVisualiser
 		static const char *faceCameraVertexSource();
 
 		static IECoreGL::ConstRenderablePtr ray();
-		static IECoreGL::ConstRenderablePtr pointRays();
+		static IECoreGL::ConstRenderablePtr pointRays( float radius );
```

Comment:
This is an API change. Let's default radius to 0 for source compatibility, and document the break.

Reply:
Addressed in 8d49e15